lazy-pages/src/process.rs

```diff
@@ -82,29 +82,30 @@ pub(crate) trait AccessHandler {
 /// program's wasm memory.
 /// 3) Charge gas for access and data loading.
 pub(crate) fn process_lazy_pages<H: AccessHandler>(
-    ctx: &mut LazyPagesExecutionContext,
+    rt_ctx: &mut LazyPagesRuntimeContext,
+    exec_ctx: &mut LazyPagesExecutionContext,
```

Comment:
Hmm, since you use now both contexts here, I cannot see the reason, why we cannot just use `ctx: LazyPagesContext` as param here and some other places. We can implement methods for it to operate with both context. But may better to do this in future PR s

Reply:
Agree